include/dqrobotics/solvers/DQ_QPOASESSolver.h

```diff
@@ -43,6 +46,9 @@ namespace DQ_robotics
         //Page 14 of https://www.coin-or.org/qpOASES/doc/3.0/manual.pdf
         int_t maximum_working_set_recalculations_;
 
+        //implimentation of the equality constraints is done via appending it and recipical of it to the inequalities
```

Comment:
@qlin960618 Please fix the typos and add some brief details about the tolerance variable. For instance:

```cpp
// Equality constraints are handled by means of appended inequality constraints within an equality tolerance threshold.
```

Reply:
@juanjqo , I see. please let me know if this is sufficient!